tests/integration/states/test_resource_group.py

```diff
@@ -64,44 +65,10 @@ def test_changes(salt_call_cli, resource_group, location, tags, connection_auth)
         name=resource_group,
         location=location,
         connection_auth=connection_auth,
-        tags=tags,
+        tags=None,  # Set tags to None
```

Comment:
**question (blocking):** does `tags=None` actually remove the tags or does it become an unmanaged item?

Reply:
it removes the tags